src/hazelcore/Statics_Exp.re

```diff
@@ -768,9 +768,10 @@ and syn_fix_holes_operand =
       | NotInVarHole =>
         let (u, u_gen) = MetaVarGen.next(u_gen);
         let reason: VarErrStatus.HoleReason.t =
-          switch (Var.is_let(x), Var.is_case(x)) {
-          | (true, _) => Keyword(Let)
-          | (_, true) => Keyword(Case)
+          switch (Var.is_let(x), Var.is_case(x), Var.is_fun(x)) {
```

Comment:
Lets use ExpandingKeyword.mk here so there is only one place we need to modify when there is a new keyword added to the system